### ISSUE TITLE
external ip does not change

### ISSUE THREAD

**Issue by krolaper** [nojsja/shadowsocks-electron#38]
My System:
  Kernel: 5.15.0-84-generic x86_64 bits: 64 compiler: gcc v: 11.4.0
    Desktop: Xfce 4.16.0 Distro: Linux Mint 21 Vanessa base: Ubuntu 22.04 jammy

Tell me what could be the problem.
Shadowsocks launched, added a server, connected, shows on the network, ping 48. But when checking my IP on the Internet, it shows a standard one, not a connected server. I ran it as root - it didn't help.

**Comment by nanCreate** [comment]
Don't you have extensions in your browser that can control the proxy? Custom proxy extensions take control and can ignore the system proxy.

**Comment by krolaper** [comment]
@nanCreate No. specially checked on several standard browsers (firefox, chromium)

**Comment by nanCreate** [comment]
Try installing any extension in the browser that is able to control the proxy. (example: [Proxy switcher](https://chromewebstore.google.com/detail/proxy-switcher-and-manage/onnfghpihccifgojkpnnncpagjcdbjod)). In the SOCKS5 settings, specify localhost and the default port 1080 (if you have not changed the default settings) and check your connection.

![image](https://github.com/nojsja/shadowsocks-electron/assets/110712717/7cc349e2-c276-4841-922c-594fe3a86926)

after entering the parameters, do not forget to save the changes by clicking on the green check mark, otherwise all changes will be lost

**Comment by krolaper** [comment]
@nanCreate This is a solution for the browser, but not for all OS that I need. What could be the problem?
